cypress-E2E-tests/cypress/support/device.commands.ts

```diff
@@ -223,15 +253,53 @@ Cypress.Commands.add("createEvent", (request) => {
       time: formattedTimestamp,
     };
     cy.apiRequest({
-      method: "POST",
-      url: "/event/events",
+      method: 'POST',
+      url: '/event/events',
       headers: {
-        Accept: "application/vnd.com.nsn.cumulocity.event+json",
+        Accept: 'application/vnd.com.nsn.cumulocity.event+json',
       },
       body: body,
       failOnStatusCode: false,
     })
-      .its("status")
-      .should("eq", 201);
+      .its('status')
+      .should('eq', 201);
+  });
+});
+
+Cypress.Commands.add('addOrRmoveDataPoint', (label, action) => {
+  cy.get(`[title*='${label}']`).parent('button').siblings('div').children(`button[data-cy="datapoint-selector-list-item--${action}-datapoint-button"]`).click();
+});
+
+Cypress.Commands.add('expandOrCollapseDataPoint', (label) => {
+  cy.get(`[title*='${label}']`).parent('button').parent('div').parent('div').siblings('div').children('button[data-cy="c8y-li--collapse-btn"]').click();
+});
+
+Cypress.Commands.add('switchDataPointToggleButton', (label) => {
+  cy.get(`[title*='${label}']`).parent('button').parent('div').parent('div').siblings('c8y-li-checkbox').click();
+});
```

Comment:
These commands can be added in asset properties widget command file.

Reply:
incorporated.